theme_anelusia/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" inherit_id="website.s_company_team_basic">
+    <!-- Team #01 -->
```

Comment:
Should this change be under `<!-- Team #01 - Img -->` ? Same for other team members

Reply:
Woops good catch I don't need to repeat the expression, it's could be part of the same xpath 👍